core/chain-configs/src/client_config.rs

```diff
@@ -499,7 +499,7 @@ impl ClientConfig {
             num_block_producer_seats,
             ttl_account_id_router: Duration::from_secs(60 * 60),
             block_fetch_horizon: 50,
-            catchup_step_period: Duration::from_millis(1),
+            catchup_step_period: Duration::from_millis(100),
```

Comment:
is this intended change?

Reply:
Yes, the previous 1ms doesn't seem to be used anywhere, and it causes the testloop based test to fire that timer every 1ms which is way too often.